test/inductor/test_fused_attention.py

```diff
@@ -631,6 +677,9 @@ class SDPAPatternRewriterCudaTests(TestSDPAPatternRewriterTemplate):
         test_sdpa_rewriter_13_cuda = functools.partialmethod(
             TestSDPAPatternRewriterTemplate._test_sdpa_rewriter_13, dtype=torch.half
         )
+        test_sdpa_rewriter_14_cuda = functools.partialmethod(
+            TestSDPAPatternRewriterTemplate._test_sdpa_rewriter_14
+        )
```

Comment:
no test for pattern 15 on cuda?

Reply:
Pattern 15 failed to match for CUDA. Added a TODO in comment.